src/portable/portable.h

```diff
@@ -12,6 +12,7 @@ typedef struct PortableMetadata {
         int fd;
         char *source;
         char *image_path;
+        char *selinux_file_con;
```

Comment:
not a fan of needless abbreviations. also, i never understood the difference between a context and a label. I'd stick to using "label" everywhere., and given that this is about files here anyway, maybe drop "file" from the name, 

tldr, maybe name this "selinux_label"?